openapi/openapi.yaml

```
@@ -769,6 +769,10 @@ paths:
     $ref: ./paths/credit-memos@{id}@timeline.yaml
   /credit-memos/{id}/timeline/{messageId}:
     $ref: ./paths/credit-memos@{id}@timeline@{messageId}.yaml
+  /credit-memos/{id}/transaction-allocations:
+    $ref: ./paths/credit-memos@{id}@transaction-allocations.yaml
+  /credit-memos/{id}/transaction-allocations/{transactionId}:
+      $ref: ./paths/credit-memos@{id}@transaction-allocations@{transactionId}.yaml
```

```suggestion
  /credit-memos/{id}/transaction-allocations:
    $ref: ./paths/credit-memos@{id}@allocations.yaml
  /credit-memos/{id}/allocations/{transactionId}:
      $ref: ./paths/credit-memos@{id}@allocations@{transactionId}.yaml
```
Can we rename like this?

---

I am afraid that this will suggest that this is for both invoice and transaction allocations. Instead of only allocations to transactions. What do you think @adamaltman?

---

I'm confused. There is no invoice allocation, so how would it suggest that?

---

There are allocations from credit memo to invoices: `/invoices/{id}/credit-memo-allocations`. Granted, it's on a different resource path, but non-specific path for transactions may be confusing about what is meant. Having "transaction" gives the idea right away.